webapp/components/rhs_comment.jsx

```diff
@@ -287,10 +287,19 @@ export default class RhsComment extends React.Component {
         );
 
         let compactClass = '';
-        let profilePicContainer = (<div className='post__img'>{profilePic}</div>);
+        const profilePicContainer = (<div className='post__img'>{profilePic}</div>);
         if (this.props.compactDisplay) {
             compactClass = 'post--compact';
-            profilePicContainer = '';
+        }
+
+        if (this.props.compactDisplay) {
```

Comment:
Can you merge this if statement with the one above?

Also can you move
```
const profilePicContainer = (<div className='post__img'>{profilePic}</div>);
```
after the if statement? I think it works because of JavaScript objects being references, but it's weird to assign a variable before you're done modifying it